include/mbedtls/check_config.h

```diff
@@ -153,7 +153,9 @@
 #endif /* not all curves accelerated */
 #endif /* some curve accelerated */
 
-#if defined(MBEDTLS_CTR_DRBG_C) && !defined(MBEDTLS_AES_C)
+#if defined(MBEDTLS_CTR_DRBG_C) && !(defined(MBEDTLS_AES_C) || \
+    (defined(MBEDTLS_USE_PSA_CRYPTO) && defined(PSA_WANT_KEY_TYPE_AES) && \
```

Comment:
I don't think `USE_PSA_CRYPTO` should be part of this. You probably mean just `PSA_CRYPTO_C` here.

Reply:
Yes, of course, I totally missed that if CTR_DRBG is used by PSA it means that PSA itself is built in the first place which itself means that `CRYPTO_C` is defined. I'll fix that

Reply:
That's not what I meant. We don't know if PSA is built - users can perfectly enable CTR-DRBG without PSA. But the feature macro that answers "is PSA built?" is called `PSA_CRYPTO_C`, not `USE_PSA_CRYPTO` (that one answers "will TLS, X.509 and PK use PSA", which of course implies PSA is built, but is not equivalent).

Reply:
Sorry, my explanation was spelled worse than I thought. I get the point anyway ;)